src/core/Microsoft.PowerToys.Settings.UI/Interop.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.PowerToys.Settings.UI
+{
+    internal static class Interop
+    {
+        public static ICoreWindowInterop GetInterop(this Windows.UI.Core.CoreWindow @this)
+        {
+            var unkIntPtr = Marshal.GetIUnknownForObject(@this);
+            try
+            {
+                var interopObj = Marshal.GetTypedObjectForIUnknown(unkIntPtr, typeof(ICoreWindowInterop)) as ICoreWindowInterop;
+                return interopObj;
+            }
+            finally
+            {
+                Marshal.Release(unkIntPtr);
+                unkIntPtr = System.IntPtr.Zero;
+            }
+        }
+
+        [DllImport("user32.dll")]
```

Comment:
Doesn't this still generate an FxCop warning allowing global access to ShowWindow?  I must be missing something.

Reply:
I'm trying to limit types of warnings into groups.  Some can cascade.